ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -343,45 +342,53 @@ else
 
 	#
 	# Check if PX4IO present and update firmware if needed.
+	# Assumption IOFW set to firware file and IO_PRESENT = no
 	#
-	if [ -f /etc/extras/px4io-v2.bin ]
-	then
-		set IO_FILE /etc/extras/px4io-v2.bin
 
-		if px4io checkcrc ${IO_FILE}
+	if [ -f $IOFW ]
+	then
+		if px4io checkcrc ${IOFW}
 		then
 			set IO_PRESENT yes
 		else
-			tune_control play -m MLL32CP8MB
-
-			if px4io start
+			# Check for the mini using build with px4io fw file
+			# but not a px4IO
+			if ver hwtypecmp V540
```

Comment:
@bkueng No we can call it. I was originally hopeful to return -ENXIO from px4io and use $? in the if. This would avoid duplicate tests and carrying the list in the init script. The hw detection in the px4io would return -ENXIO instead of 1. Alas it did not work and I did not pursue it. 

I changed it to you suggestions.